blockreassurance.php

```diff
@@ -539,8 +545,10 @@ protected function addJsDefList()
             'psr_text_color' => Configuration::get('PSR_TEXT_COLOR'),
             'psr_controller_block_url' => $this->context->link->getAdminLink('AdminBlockListing'),
             'psr_controller_block' => 'AdminBlockListing',
+            'psr_lang' => (int) Configuration::get('PS_LANG_DEFAULT'),
             'block_updated' => $this->trans('Block updated', array(), 'Modules.Blockreassurance.Admin'),
             'active_error' => $this->trans('Oops... looks like an error occurred', array(), 'Modules.Blockreassurance.Admin'),
+            'min_field_error' => $this->trans('The Title field is required at least in your default language.', array(), 'Modules.Blockreassurance.Admin'),
```

Comment:
Hey, would it be possible to write _The field %field_name% is required at least in your default language._ (localized in _Admin.Notifications.Error_) instead? We already have this string to express the same thing. ;-)

Reply:
:ok_hand: